packages/smooth_app/pubspec.lock

```diff
@@ -569,20 +562,6 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.0"
-  google_mlkit_barcode_scanning:
-    dependency: "direct main"
-    description:
-      name: google_mlkit_barcode_scanning
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "0.3.0"
-  google_mlkit_commons:
-    dependency: transitive
-    description:
-      name: google_mlkit_commons
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "0.2.0"
```

Comment:
This shouldn't be gone right? Just not `direct main` anymore

Reply:
This dependency is now on another module. Not on smooth_app anymore

Reply:
Ohh yes right 😄